core/stage.go

```diff
@@ -120,9 +120,9 @@ type Stage struct { //types:add -setters
 	// [WindowStage]s take up the entire window they are created in.
 	FullWindow bool
 
-	// For NewWindow stages, use the actual contents size, instead of
+	// For NewWindow stages, use the actual content size, instead of
 	// enforcing a minimum size as a function of the total screen size.
-	ContentsSize bool
+	ContentSize bool
```

Comment:
I can see your point for the other things that they return the content size, but this is about whether you should size to content. `ContentSize bool` doesn't really make sense; I much prefer `SizeToContent`.

Reply:
New plan: `UseMinSize`  -- set to true by default.  also apply to dialogs.